configure

```diff
@@ -451,7 +451,7 @@ if [ "${LIB_DEPRECATE_NAME}" ]; then
     fi
 fi
 DRIVER_DIR=${DRIVER_DIR:-${ARCH_DIR}/${DRIVER}}
-CFLAGS="$CPUFLAGS -Ofast -Wall -pthread $CFLAGS"
+CFLAGS="$CPUFLAGS -Ofast -Wall $CFLAGS"
```

Comment:
A note from pigpio docs:
>To compile, link, and run a C program
>```
>gcc -Wall -pthread -o foobar foobar.c -lpigpio -lrt
>sudo ./foobar
>```

So, I think `pthread` is still an implicit dep.

Reply:
Re-added `-pthread`